ranger_base/include/ranger_base/ranger_params.hpp

```diff
@@ -52,10 +52,10 @@ struct RangerMiniV2Params {
   static constexpr double max_angular_speed = 0.7853;  // in rad/s
   static constexpr double max_speed_cmd = 10.0;        // in rad/s
 
-  static constexpr double max_steer_angle_central = 0.4782;  //~= 27.40 degree
+  static constexpr double max_steer_angle_ackermann = 0.6981;   // 40 degree
```

Comment:
These Values have been changed upstream. Please check if this or the one upstream is valid?